linux-tkg-config/6.5/minimal-modprobed.db

```diff
@@ -352,16 +357,17 @@ nf_nat_irc
 nf_nat_pptp
 nf_nat_sip
 nf_nat_tftp
+nf_reject_ipv4
+nf_reject_ipv6
+nf_tables
 nfnetlink
 nfnetlink_log
 nfnetlink_queue
-nf_reject_ipv4
-nf_reject_ipv6
 nfs
 nfsv4
-nf_tables
 nft_chain_nat
 nft_compat
+nft_counter
```

Comment:
This module also doesn't seem to be in-tree, as I get a log about `nft_counter config not found!!`. I'll remove this one too then.